src/ansys/aedt/core/hfss.py

```diff
@@ -1428,41 +1509,33 @@ def create_sbr_antenna(
             parameters_defaults["Power Fraction"] = "0.95"
         if is_array:
             parameters_defaults["Is Parametric Array"] = True
-            parameters_defaults["Array Element Type"] = self.SBRAntennaDefaults.default_type_id[antenna_type]
-            parameters_defaults["Array Element Angle Phi"] = ("0deg",)
-            parameters_defaults["Array Element Angle Theta"] = ("0deg",)
-            parameters_defaults["Array Element Offset X"] = "0meter"
-            parameters_defaults["Array Element Offset Y"] = "0meter"
-            parameters_defaults["Array Element Offset Z"] = "0meter"
-            parameters_defaults["Array Element Conformance Type"] = 0
-            parameters_defaults["Array Element Conformance Type"] = 0
-            parameters_defaults["Array Element Conformance Type"] = 0
-            parameters_defaults["Array Element Conform Orientation"] = False
-            parameters_defaults["Array Design Frequency"] = "1GHz"
-            parameters_defaults["Array Layout Type"] = 1
-            parameters_defaults["Array Specify Design In Wavelength"] = True
-            parameters_defaults["Array Element Num"] = 5
-            parameters_defaults["Array Length"] = "1meter"
-            parameters_defaults["Array Width"] = "1meter"
-            parameters_defaults["Array Length Spacing"] = "0.1meter"
-            parameters_defaults["Array Width Spacing"] = "0.1meter"
-            parameters_defaults["Array Length In Wavelength"] = "3"
-            parameters_defaults["Array Width In Wavelength"] = "4"
-            parameters_defaults["Array Length Spacing In Wavelength"] = "0.5"
-            parameters_defaults["Array Stagger Type"] = 0
-            parameters_defaults["Array Stagger Angle"] = "0deg"
-            parameters_defaults["Array Symmetry Type"] = 0
-            parameters_defaults["Array Weight Type"] = 3
-            parameters_defaults["Array Beam Angle Theta"] = "0deg"
-            parameters_defaults["Array Weight Edge TaperX"] = -200
-            parameters_defaults["Array Weight Edge TaperY"] = -200
-            parameters_defaults["Array Weight Cosine Exp"] = 1
-            parameters_defaults["Array Differential Pattern Type"] = 0
-            if is_array:
-                name = generate_unique_name("pAntArray")
+            parameters_defaults["Sheet"] = -1
```

Comment:
Just wondering why `-1` is used as default. Wouldn't it be better to set it to `None` ?

Reply:
Because the AEDT API needs a -1, or at least this is what is recorded, and in the API documentation there is no more information.